### PR TITLE
added section on exploring new and unfamiliar content

### PR DESCRIPTION
**Pull request by kierisi**
I'm not sure if this should live in Chapter 5 or not, but I've added content that's ready for review! this is based on the "Learning to Learn" post that I wrote regarding the `coalesce()` function, and should provide guidance for readers when they encounter new or unfamiliar content.

### CODE REVIEW

File: 05-foundational-skills_1.Rmd [hunk 1]
```diff
@@ -145,4 +143,45 @@ dataedu::course_data
 
 ## Steps for working through new and unfamiliar content
 
-Need to add
\ No newline at end of file
+One skill that all great educators possess is the ability to ask questions. 
+Asking the right questions at the right time of the learners in your classroom can facilitate understanding, uncover misconceptions, and indicate whether or not learners have mastered the material.
```
Comment (jrosen48) [inline 05-foundational-skills_1.Rmd]: love this

File: 05-foundational-skills_1.Rmd [hunk 2]
```diff
@@ -145,4 +143,45 @@ dataedu::course_data
 
 ## Steps for working through new and unfamiliar content
 
-Need to add
\ No newline at end of file
+One skill that all great educators possess is the ability to ask questions.
```
Comment (jrosen48) [inline 05-foundational-skills_1.Rmd]: super duper minor, maybe soften this just a little bit - maybe something like: "A sure sign of a great educator is the ability to ask questions."?

File: 05-foundational-skills_1.Rmd [hunk 3]
```diff
@@ -145,4 +143,45 @@ dataedu::course_data
...
+**Activate prior knowledge**
+You've been reading through a tutorial and have come across the `coalesce()` function. 
+Take a moment to think through the following questions:
+
+* What does the word "coalesce" mean?
+* Have you ever seen the `coalesce()` function before? If so, in what context?
+
+**Look for context clues**
+
+* Read a couple of lines of code both above and below where the `coalesce()` function appears - are there any clues as to what this function might do?
```
Comment (jrosen48) [inline 05-foundational-skills_1.Rmd]: wondering what you think about adding 2-3 lines of code here - with the middle line including the `coalesce()` function? Perhaps these could be inspired from some use of `coalesce()` in the wild?

File: 05-foundational-skills_1.Rmd [hunk 4]
```diff
@@ -145,4 +143,45 @@ dataedu::course_data
...
+
+**Check the docs**
+
+* What information is available in the Help documentation?
+* Are there any examples from the Help documentation that seem similar to what you're trying to accomplish? If so, choose one.
```
Comment (jrosen48) [inline 05-foundational-skills_1.Rmd]: maybe show one example from the docs here? not sure

File: 05-foundational-skills_1.Rmd [hunk 5]
```diff
@@ -145,4 +143,45 @@ dataedu::course_data
...
+
+**Find the limits**
+Work through examples in the Help documentation (or examples that you've found online) and test the limits.
+Testing the limits is a way of understanding the code by seeing how it handles different situations.
```
Comment (jrosen48) [inline 05-foundational-skills_1.Rmd]: love this - reminds me of student uses of simulations, when it's often pedagogically beneficial for them to crank the simulation up (or down) to its limits to see how it behaves!

File: 05-foundational-skills_1.Rmd [hunk 6]
```diff
@@ -145,4 +143,45 @@ dataedu::course_data
...
+Ultimately what you're doing is recognizing a pattern, developing a hypothesis, and testing whether or not that hypothesis is true.
```
Comment (jrosen48) [inline 05-foundational-skills_1.Rmd]: nice, love this, too!

File: 05-foundational-skills_1.Rmd [hunk 7]
```diff
@@ -145,4 +143,45 @@ dataedu::course_data
...
+* Substituting different data types
+* What happens if you introduce `NA` values?
+* Is the order of values important?
+
+**Test (and refine) your understanding**
+Take a moment to think through whether or not you could explain what you've just learned to someone else.
+Imagine the questions that they might ask of you, and try to answer them.
+If you can't answer them, dig deeper into the documentation, online forums, or even in testing your own knowledge, until you feel like you can! 
+
+You won't necessarily have the time (or interest!) in doing this for each new or unfamiliar piece of content that you come across, but we hope that this provides you with a starting framework for furthering your understanding when you _do_ come across content that you want to explore a bit more in-depth!
```
Comment (jrosen48) [inline 05-foundational-skills_1.Rmd]: this is great

File: 05-foundational-skills_1.Rmd (hunk 2 above)
Comment (kierisi) [reply 05-foundational-skills_1.Rmd]: great point - done!

File: 05-foundational-skills_1.Rmd (hunk 4 above)
Comment (kierisi) [reply 05-foundational-skills_1.Rmd]: excellent idea - rather than saying "you see it in context," _showing_ it in context!

**Pull request merged**